src/lnav_commands.cc

```diff
@@ -454,6 +454,10 @@ static string com_save_to(string cmdline, vector<string> &args)
         return "";
     }
 
+    if (lnav_data.ld_secure_mode) {
+        return args[0] + ": unavailable in secure mode";
```

Comment:
Start the message with "error: ", so something like:

  string("error: ") + args[0] + " -- is unavailable in secure mode";

Tagging with "error:" will cause it to be printed to stderr in headless mode.